crates/sui-cost-constants/src/constants.rs

```diff
@@ -56,7 +57,7 @@ pub const OBJ_METADATA_COST_REFUNDABLE: u64 = 100;
 // Consensus costs: costs for TXes that use shared object
 //
 // Flat cost for consensus transactions
-pub const CONSENSUS_COST: u64 = 1_000;
+pub const CONSENSUS_COST: u64 = 100_000;
```

Comment:
For my own knowledge, how were these numbers determined?

Reply:
@tnowacki  these are the numbers we've always had. Once all this logic goes in, we will decide on the actual numbers.